assets/js/vue/components/search/SearchForm.vue

```diff
@@ -66,31 +66,31 @@
                         </div>
                     </b-form>
                 </div>
-            </div>
```

Comment:
Where's this div open?

Reply:
I cross checked, i don't see any orphan div

Reply:
just odd how github displayed it I guess